docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md

```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
 
 ### 1. Add your script to the Data Platform Project using the GitHub UI
 
-:::note
-If your Glue job was created using the Glue Studio UI,
-you can find the automatically generated script by navigating to the `Script` tab when viewing your job in Glue Studio.
+1. Navigate to the [jobs directory][jobs-directory] in the Data Platform Project and open your department folder.
+1. Click `Add file` and then `Create new file`.
+1. Name your file.
+   This should be the same name you gave your job in Glue Studio, but should be with all **lowercase letters**, **words separated by underscores**, and with the extension **`.py`** at the end.
+   e.g. `address_cleaning_housing_repairs.py`.
+   - _Remember this name as you will need it for the following section._
+1. Open your Glue job in Glue Studio and copy the auto generated **Python** script by navigating to the `Script` tab.
+
+_Note: This is only applicable if your Glue job was created using the Glue Studio UI._
+
+1. Navigate back to the new script file you created in the GitHub UI (or existing script file, if you are updating your script), and paste your script.
+1. Submit your changes by referring to the [Committing changes][committing-changes] section of the **Using Github** guide
+   The Data Platform team needs to approve any changes to the code, so your change won't happen automatically.
+
+:::important Updating your Glue job script
+If you need to update your Glue job script in the future, you must follow steps 4-6 to have those changes deployed to the Data Platform Project.
 :::
 
-- Open the [scripts directory][scripts-directory] in the Data Platform Project and navigate to your department folder
-- Click `Add file` and then `Create new file`
-- Name your file, using underscores as separators e.g. `address_cleaning_housing_repairs`, and then paste your code in field below
-- Follow [the instructions here][committing-changes] to commit your new file to the project
-
 ### 2. Copy an existing Glue job module block or use the example below and populate the necessary variables
 
-Once the Pull Request to add your script has been approved, and you have confirmed that your script is in the Data Platform Project,
-you can proceed with this step.
-In this step, you will be creating a module which will deploy your Glue job (and crawler) to the Data Platform Project.
+Once the Pull Request to add or update your script has been approved, and you have confirmed that your script is in the Data Platform Project,
```

Comment:
I guess they don't need to wait for this in order to continue to making a second pull request?